example_data/sample_bundle.json

```diff
@@ -108,45 +126,54 @@
             {
               "system": "http://snomed.info/sct",
               "code": "122555007",
-              "display": "Venus blood specimen"
+              "display": "blood"
             }
           ]
         },
         "subject": {
           "type": "Patient",
-          "reference": "Patient/37176e84-d977-e993-3c49-d76fcfc6e625"
+          "reference": "Patient?_id=gosh-redcap-40388914"
         }
       },
       "request": {
-        "method": "POST",
-        "url": "Specimen"
+        "method": "PUT",
```

Comment:
wont comment on ones further but I had thought that it should be POST for a bundle

Reply:
I tried it and it worked on both our synthetic FHIR server and even the Hapi FHIR I was working with